ambari-metrics/ambari-metrics-timelineservice/src/main/java/org/apache/hadoop/yarn/server/applicationhistoryservice/metrics/timeline/PhoenixHBaseAccessor.java

```diff
@@ -337,25 +337,36 @@ public void commitMetrics(Collection<TimelineMetrics> timelineMetricsCollection)
           double[] aggregates = AggregatorUtils.calculateAggregates(
                   metric.getMetricValues());
 
-          byte[] uuid = metadataManagerInstance.getUuid(metric);
-          if (uuid == null) {
-            LOG.error("Error computing UUID for metric. Cannot write metrics : " + metric.toString());
-            continue;
+          if (aggregates[3] != 0.0) {
+            byte[] uuid = metadataManagerInstance.getUuid(metric);
+            if (uuid == null) {
+              LOG.error("Error computing UUID for metric. Cannot write metrics : " + metric.toString());
```

Comment:
Should be a WARN IMO. Can we open Jira to blacklist such a metric?

Reply:
Yes I agree. I am also working on a follow up JIRA to clean up UUID computation. Will handle it in that one.